cmd/version.go

```diff
@@ -0,0 +1,27 @@
+package cmd
+
+import (
+	"log"
+
+	"github.com/openfga/openfga/internal/build"
+	"github.com/spf13/cobra"
+)
+
+// NewVersionCommand returns the command to get openfga version
+func NewVersionCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "Return the OpenFGA version",
+		Long:  "Return the OpenFGA version.",
+		RunE:  version,
+		Args:  cobra.NoArgs,
+	}
+
+	return cmd
+}
+
+// print out the built version
+func version(_ *cobra.Command, _ []string) error {
+	log.Printf("OpenFGA Version %s Date %s commit id %s ", build.Version, build.Date, build.Commit)
```

Comment:
Minor nit pick, but do you mind if we format this like

> OpenFGA version 'v0.3.6' built from '3e8facb' on '2023-02-19T20:54:52Z'

Reply:
done